File: GIT-USERS/TOM-Lambda/CSEU3-SORTING-GP/book.py
class Book:
    def __init__(self, title, author, genre):
        self.title = title
        self.author = author
        self.genre = genre

    def __str__(self):
<<<<<<< HEAD
        return f'{self.genre}: {self.title} by {self.author}'
=======
        return f"{self.genre}: {self.title} by {self.author}"
>>>>>>> 23fb4d348bb9c7b7b370cb2afcd785793e3816ea

File: GIT-USERS/ashishdotme/dotnetcore-api-starter/Starter.API/Data/Models/Book.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Starter.API.Data.Models
{
    public class Book
    {
        public Guid Id { get; set; }
        [Required]

        public string Title { get; set; }
        public string Author { get; set; }
        public string Description { get; set; }
    }
}
